ldk/javascript/src/types/oliveHelps/index.d.ts

```diff
@@ -248,6 +248,7 @@ declare namespace OliveHelps {
     Select = 'select',
     Telephone = 'telephone',
     TextInput = 'textInput',
+    Form = 'form',
```

Comment:
I thought this d.ts file is representation of what we could communicate to/from goja runtime. And since Form is the LDK only concept, does it needs to be here?

Reply:
This is a good point and I think that including it here is incorrect because it implies that Goja can meaningfully do something with these fields and types, and it cannot. Best case it will ignore. Worst case it will error out.

Reply:
I do not disagree. I will have to create some kind of mapper to go from our LDK `NewWhisper` type to the `NewWhisper` `d.ts` contract with OH.